@xen-orchestra/xapi/host.mjs

```diff
@@ -120,6 +127,48 @@ class Host {
     await this.callAsync('host.reboot', ref)
     await waitAgentRestart(this, ref, agentStartTime)
   }
+
+  async getIpmiSensors(ref, { cache } = {}) {
+    let productName
+
+    const productNameCacheKey = this.computeCacheKey('host', ref, 'system-product-name')
+    if (cache?.has(productNameCacheKey)) {
+      productName = cache.get(productNameCacheKey)
+    } else {
+      productName = (await this.getField('host', ref, 'bios_strings'))['system-product-name']?.toLowerCase()
```

Comment:
Use `call` and it will handle the cache for you.

```suggestion
      productName = (await this.call(cache, 'host.get_bios_strings', ref))['system-product-name']?.toLowerCase()
```